node_modules/ioslib/lib/certs.js

```diff
@@ -166,7 +166,7 @@ function detect(options, callback) {
 								if (!code) {
 									var tmp = [];
 									parseCerts(out, tmp);
-									results.certs.wwdr = tmp.length && tmp[0].invalid === false;
+									results.certs.wwdr = results.certs.wwdr || (tmp.length && tmp[0].invalid === false);
```

Comment:
Mmm, right... but since the tasks for all keychains run in parallel the same task for another keychain might have set it to true between the test in 163 and when the subprocess calls back 166.